commit ff435e359364b968d14bb4c96b5698712d9cea90
Author: Matthias Reichl <hias@horus.com>
Date:   Fri Dec 18 20:53:43 2020 +0100

    vc4_hdmi: Reduce max pixel rate to hide 4k modes
    
    162MHz limits to 1920x1200@60

diff --git a/drivers/gpu/drm/vc4/vc4_hdmi.c b/drivers/gpu/drm/vc4/vc4_hdmi.c
index 2386c7303a215..561d3fa3bb926 100644
--- a/drivers/gpu/drm/vc4/vc4_hdmi.c
+++ b/drivers/gpu/drm/vc4/vc4_hdmi.c
@@ -404,7 +404,7 @@ static void hdmi_codec_eld_chmap(struct vc4_hdmi *vc4_hdmi)
 		vc4_hdmi->audio.chmap = hdmi_codec_stereo_chmaps;
 }
 
-#define HDMI_14_MAX_TMDS_CLK   (340 * 1000 * 1000)
+#define HDMI_14_MAX_TMDS_CLK   (162 * 1000 * 1000)
 
 static int vc4_hdmi_debugfs_regs(struct seq_file *m, void *unused)
 {
